app/controllers/posts_controller.rb

```diff
@@ -17,8 +17,8 @@ def new
   end
 
   def create
-    @post = Post.new(post_params.merge(user: current_user))
-    if @post.save
+    post = PostCreation.call(current_user, post_params)
```

Comment:
what will be in post after call ?

Reply:
after call returns instance a Post or false.I have an error in business logic

Reply:
why do you name variable as post if it will contain true or false? maybe just result?